frontends/mit-learn/src/page-components/ManageListDialogs/ManageListDialogs.tsx

```diff
@@ -191,6 +183,11 @@ const UpsertLearningPathDialog = NiceModal.create(
           value={formik.values.published}
           onChange={(e) => formik.setFieldValue(e.name, e.value)}
         />
+        {mutation.isError && !formik.isSubmitting && (
+          <Alert severity="error">
+            There was a problem saving your list. Please try again later.
+          </Alert>
+        )}
```

Comment:
Here's what it looks like now. Same as before.

<img width="692" alt="Screenshot 2024-08-29 at 1 52 42 PM" src="https://github.com/user-attachments/assets/2fc60e83-8388-410b-9485-e34341215ba0">